app/javascript/src/ActiveDocs/components/ActiveDocsSpec.jsx

```diff
@@ -1,18 +1,26 @@
 // @flow
 
 import React from 'react'
-
 import { createReactWrapper } from 'utilities/createReactWrapper'
 import SwaggerUI from 'swagger-ui-react'
-
 import 'swagger-ui-react/swagger-ui.css'
+import type { SwaggerResponse } from 'Types/SwaggerTypes'
 
 type ActiveDocsSpecProps = {
-  url: string
+ accountDataUrl: string,
+ autocompleteOAS3: (response: SwaggerResponse, accountDataUrl: string) => Promise<SwaggerResponse>,
+ url: string,
 }
 
-const ActiveDocsSpec = ({ url }: ActiveDocsSpecProps) => <SwaggerUI url={url} />
+const ActiveDocsSpec = ({ url, accountDataUrl, autocompleteOAS3 }: ActiveDocsSpecProps) => (
+  <SwaggerUI
+    url={url}
+    responseInterceptor={(response) => autocompleteOAS3(response, accountDataUrl)}
```

Comment:
Looks like `accountDataUrl` and `autocompleteOAS3` don't have to be passed. I would reduce props by passing `responseInterceptor` directly. This way `ActiveDocsSpec` and `SwaggerUI` maintain the same signature.